walkthroughs/howto-timeout-policy/deploy.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```

Comment:
Instead of one large "deploy" script, how about breaking this up into VPC, ECR repos/images, mesh, ECS infra?

The catch to having these all in one script is, if any individual step fails, you have to start over. :(

Reply:
Discussed offline - we'll do this as a follow-up.